openwisp_notifications/models.py

```diff
@@ -17,6 +19,12 @@
 
 class Notification(UUIDModel, AbstractNotification):
     COUNT_CACHE_KEY = 'ow2-unread-notifications-{0}'
+    notification_type = models.CharField(
```

Comment:
This would result in `Notification.notification_type`, which is redundant, I'd rename it to just `type`, don't worry about the base `type()` function having the same name, it's ok if we do it deliberately and we know what we are doing

Reply:
Thanks, that was my only concern to do it like this. I will change it to `type`.